modules/exploits/linux/misc/zyxel_multiple_devices_zhttp_lan_rce.rb

```diff
@@ -0,0 +1,240 @@
+        'DefaultOptions' => {
+          'PAYLOAD' => 'linux/armle/meterpreter/reverse_tcp',
+          'WfsDelay' => 15
+        },
+        'Targets' => [
+          [ 'DX3301-T0 <= V5.50(ABVY.3)C0', {} ],
+          [ 'DX5401-B0 <= V5.17(ABYO.1)C0', {} ],
+          [ 'EMG3525-T50B (EMEA) <= V5.50(ABPM.6)C0', {} ],
+          [ 'EMG3525-T50B (S. America) <= V5.50(ABSL.0)b', {} ],
+          [ 'EMG5523-T50B (EMEA) <= V5.50(ABPM.6)C0', {} ],
+          [ 'EMG5523-T50B (S. America) <= V5.50(ABSL.0)b', {} ],
+          [ 'EMG5723-T50K <= V5.50(ABOM.7)C0', {} ],
+          [ 'EX3301-T0 <= V5.50(ABVY.3)C0', {} ],
+          [ 'EX5401-B0 <= V5.17(ABYO.1)C0', {} ],
+          [ 'EX5501-B0 <= V5.17(ABRY.2)C0', {} ],
+          [ 'LTE3301-PLUS <= V1.00(ABQU.3)C0', {} ],
+          [ 'LTE7240-M403 <= V2.00(ABMG.4)C0', {} ],
+          [ 'VMG1312-T20B <= V5.50(ABSB.5)C0', {} ],
+          [ 'VMG3625-T50B <= V5.50(ABPM.6)C0', {} ],
+          [ 'VMG3927-B50A <= V5.17(ABMT.6)C0', {} ],
+          [ 'VMG3927-B60A <= V5.17(ABMT.6)C0', {} ],
+          [ 'VMG3927-T50K <= V5.50(ABOM.7)C0', {} ],
+          [ 'VMG4005-B50A <= V5.15(ABQA.2)C0', {} ],
+          [ 'VMG8623-T50B <= V5.50(ABPM.6)C0', {} ],
+          [ 'VMG8825-B50A <= V5.17(ABMT.6)C0', {} ],
+          [ 'VMG8825-B50B <= V5.17(ABNY.7)C0', {} ],
+          [ 'VMG8825-B60A <= V5.17(ABMT.6)C0', {} ],
+          [ 'VMG8825-B60B <= V5.17(ABNY.7)C0', {} ],
+          [ 'VMG8825-T50K <= V5.50(ABOM.7)C0', {} ],
+          [ 'XMG3927-B50A <= V5.17(ABMT.6)C0', {} ],
+          [ 'XMG8825-B50A <= V5.17(ABMT.6)C0 ', {} ],
+          [ 'VPN2S <= V1.20(ABLN.2)_00210319C1', {} ],
+          [ 'AX7501-B0 <= V5.17(ABPC.1)C0', {} ],
+          [ 'EP240P <= V5.40(ABVH.1)C0', {} ],
+          [ 'PMG5317-T20B <= V5.40(ABKI.4)C0', {} ],
+          [ 'PMG5617GA <= V5.40(ABNA.2)C0', {} ],
+          [ 'PMG5622GA <= V5.40(ABNB.2)C0', {} ],
+          [ 'WX3100-T0 <= V5.50(ABVL.1)C0', {} ],
+          [ 'WX3401-B0 <= V5.17(ABVE.1)C0', {} ],
+          [ 'WSQ50 (Multy X) <= V2.20(ABKJ.7)C0', {} ],
+          [ 'WSQ60 (Multy Plus) <= V2.20(ABND.8)C0', {} ],
+          [ 'AMG1302-T11C <= EOL', {} ],
+          [ 'VMG3925-B10C <= EOL', {} ],
+          [ 'VMG8924-B10D <= EOL', {} ],
+          [ 'VMG1312-B10D <= EOL', {} ],
+          [ 'VMG3312-T20A <= EOL', {} ],
+          [ 'VMG3625-T20A <= EOL', {} ],
+          [ 'VMG3925-B10B <= EOL', {} ],
+          [ 'VMG3925-B10C <= EOL', {} ],
+          [ 'VMG3925-B30C <= EOL', {} ],
+          [ 'VMG3926-B10A <= EOL', {} ],
+          [ 'VMG5313-B10B <= EOL', {} ],
+          [ 'VMG5313-B30B <= EOL', {} ],
+          [ 'VMG8623-T50A <= EOL', {} ],
+          [ 'VMG8823-B10B <= EOL', {} ],
+          [ 'VMG8823-B30B <= EOL', {} ],
+          [ 'VMG8823-B50B <= EOL', {} ],
+          [ 'VMG8823-B60B <= EOL', {} ],
+          [ 'VMG8924-B10D <= EOL', {} ],
+          [ 'VMG8924-B30D <= EOL', {} ],
+          [ 'PMG5317-T20A <= EOL', {} ],
```

Comment:
These do not really add value for module execution, while they would be possible use in a `search` the user is unlikely to even know the model or firmware info.  Since the target does change how the module will execute I would suggest only the standard `Automatic` or `Zyxel device` is really needed here.

```suggestion
        'Targets' => [
          [ 'Zyxel Device', {} ],
```